code/game/objects/effects/anomalies.dm

```diff
@@ -191,8 +198,11 @@
 
 /obj/effect/anomaly/bluespace/anomalyEffect()
 	..()
-	for(var/mob/living/M in range(1, src))
+	for(var/mob/living/M in range(3, src))
 		do_teleport(M, locate(M.x, M.y, M.z), 4)
+	for(var/obj/item/O in range (3, src))
+		if(!O.anchored && O.invisibility == 0 && prob(50))
+			do_teleport(O, locate(O.x, O.y, O.z), 6)
```

Comment:
please test that this is actually diffusible.
teleporting everyone in 3 tiles away every 2 seconds seems a bit much

Reply:
So I tested it locally cause I was curious: I had no issues scanning the anomaly.
Might even be too easy and safe, unless the anomaly is close to somewhere dangerous.

Reply:
Able to scan it 4 times in a row, you just need to time it. Still a bit easy, but it is bluespace, not much one can do.